src/types.ts

```diff
@@ -357,6 +357,7 @@ export type GenomeBrowserType = {
   goto: (left: number, right: number) => void;
   jump: (location: string) => void;
   set_y: (y: number) => void;
+  switch: (path: string[], value: string[] | boolean | null) => void;
   set_switch: (path: string[]) => void;
```

Comment:
Should this be removed?

Reply:
Yes think we don't need both `set_switch` and `clear_switch` anymore